.github/actions/deps-install/action.yml

```diff
@@ -48,3 +48,18 @@ runs:
       run: |
         sudo apt-get update -y
         sudo apt-get install -y libudev-dev
+
+    - name: Install python3
+      uses: actions/setup-python@v5
+      env:
+        TMP: ${{ inputs.temp || runner.temp }}
```

Comment:
You don't need to declare this env variable.

Reply:
It's leftover copypasta, thanks!